app/styles/oss-components.less

```diff
@@ -21,6 +21,7 @@
 @import 'modal-dialog';
 @import 'alert';
 @import 'country-selector';
+@import 'tooltip';
```

Comment:
let's add it to the right folder given it's something new? (atoms or molecules?)